aws_xray_sdk/core/serverless_context.py

```diff
@@ -0,0 +1,131 @@
+import os
+import logging
+
+from .models.facade_segment import FacadeSegment
+from .models.segment import Segment
+from .models.mimic_segment import MimicSegment
+from .context import CXT_MISSING_STRATEGY_KEY
+from .lambda_launcher import LambdaContext
+from .context import Context
+
+
+log = logging.getLogger(__name__)
+
+
+class ServerlessContext(LambdaContext):
```

Comment:
Again this class name is too generic and will not work well in the long term. It's tailored for web framework + lambda user case given the limitation we currently have.

Reply:
Okay, I think ServerlessLambdaContext makes more sense then. Any thoughts?